source/uwp/AdaptiveCardTestApp/Views/DiffView.xaml.cs

```diff
@@ -18,8 +19,107 @@ public DiffView()
             this.InitializeComponent();
         }
 
+        // a simplistic JSON formatter for use in displaying for diffview
+        private string FormatJson(string json)
```

Comment:
>FormatJson [](start = 23, length = 10)

I used JsonConvert in my visualizer updates for my templating prototype to achieve something similar without having to have hand rolled code for it. Not sure if this is the best approach but it got the job done:

dynamic parsedJson = JsonConvert.DeserializeObject(templateResult.ToString());
string TemplatedJson = JsonConvert.SerializeObject(parsedJson, Formatting.Indented);

Reply:
I'll give that a try. I was without internet access when I wrote this :)

---
In reply to: [257412073](https://github.com/Microsoft/AdaptiveCards/pull/2391#discussion_r257412073) [](ancestors = 257412073)

Reply:
works like a champ!

---
In reply to: [257415073](https://github.com/Microsoft/AdaptiveCards/pull/2391#discussion_r257415073) [](ancestors = 257415073,257412073)